index.html

```diff
@@ -114,6 +114,35 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+    <!-- ________ *Neobit* Contributor card START ________  -->
+<div class="card">
+          <p class="name">grixxxx</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/grixxxx" target="_blank">grixxxx</a>
+              <i class="fab fa-telegram"></i>
+            <a href="t.me/grixonly" target="_blank">grixxxx</a>
+              <i class="fab fa-linkedin"></i>
+            <a href="https://www.linkedin.com/in/delmer-john-ballesteros-b8b683252/" target="_blank">Delmer John Ballesteros</a>
+          </p>
+          <p class="about">Front-End Developer | Student</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://frontendmasters.com/course" target="_blank" title="First Resource">frontend Developer Roadmap</a>
```

Comment:
It's not a roadmap but a bunch of courses. Kindly check the url or fix the title. Thanks!
```suggestion
                <a href="https://frontendmasters.com/course" target="_blank" title="First Resource">frontend Developer Roadmap</a>
```